CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## 1.2.0
```

Comment:
Can we revert this and all other instances of version bumps? I can do that in a separate commit.

```suggestion
## x.x.x
```